components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
+
+tracer:
+
+$$
+\frac{d{\tilde h}_k \varphi_k  }{dt} 
++
+   \nabla \cdot \left({\tilde h}_k \varphi_k {\bf u}_k \right) 
+ + \left[ \varphi {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_k^{\text{top}}} 
+ - \left[ \varphi {\tilde w}_{tr} \right]_{{\tilde z}={\tilde z}_{k+1}^{\text{bot}}}
+= 0
+$$ (layer-tracer)
+
+### Layered Momentum
+
+We now derive the horizontal momentum equation in our non-Boussinesq, hydrostatic framework, following the same finite-volume approach used for mass and tracer conservation. We work with a pseudo-height vertical coordinate $\tilde{z}$ as defined in [Pseudo-Height Coordinate Section](pseudo-height).
+
+We begin by specifying the forces in the full three-dimensional momentum equation [](#continuous-momentum),
+
+$$
+\frac{d}{dt} \int_{V(t)} \rho\,  {\bf v} \, dV
++ \int_{\partial V(t)}\rho\, {\bf v} \left({\bf v} - {\bf v}_r \right) \cdot {\bf n} \, dA
+= \mathbf{F}_\text{total}[V(t)],
+$$ (continuous-momentum2)
+
+with total forces given by:
+
+$$
+\mathbf{F}_\text{total}[V(t)] =
+- \int_{V(t)} \rho\, \mathbf{f} \times \mathbf{u} \, dV
+- \int_{V(t)} \rho\, \nabla_{3D} \Phi \, dV
+- \int_{\partial V(t)} p \, \mathbf{n} \, dA
++ \int_{\partial V(t)} \boldsymbol{\tau} \cdot \mathbf{n} \, dA
+$$ (momentum-Ftotal)
+
+Each term on the right-hand side corresponds to a physically distinct force acting on the fluid within the control volume:
+
+- The first term is the **Coriolis force**, where $ \mathbf{f} $ is the vector Coriolis parameter (e.g., $ f \hat{\mathbf{z}} $ on the sphere).
+- The second term represents the **gravitational force**, expressed in terms of the gradient of the gravitational potential $ \Phi(x, y, z, t) $, which may include effects such as tides and self-attraction and loading.
+- The third term is the **pressure force**, which acts on the boundary surfaces and is naturally expressed as a surface integral. It gives rise to both horizontal pressure gradients and contributions from sloping surfaces.
+- The fourth term represents the surface stresses. They include **horizontal stress forces** across the vertical sides of the control volume, such as those due to lateral friction or subgrid momentum transfer.
+They also include **vertical stress forces** (e.g., wind stress at the ocean surface and bottom drag at the seafloor), projected into horizontal momentum and acting across the top and bottom surfaces.
+
+#### Pressure Term
+
+The pressure force term may be converted from the boundary to the interior with Gauss' divergence theorem (see [Kundu et al. 2016](https://www.amazon.com/dp/012405935X/) p. 119),
+
+$$
+- \int_{\partial V(t)} p \, \mathbf{n} \, dA 
+= - \int_{V(t)} \nabla_{3D} p \, dV .
+$$ (gradp-Gauss)
+
+Considering only the horizontal components, we have
+
+$$
+ - \int_{V(t)} \nabla_{\perp} p \, dV
+= - \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} 
+  \nabla_{\perp} p \, dz \, dA
+= - \int_{A}   \overline{\nabla_{\perp} p}^{z}  \, dA.
+$$ (gradp-h)
+
+#### Stress Term
```

Comment:
This section also uses the $\overline{X}^{z}$ notation and should instead use $\overline{1/\rho X}^{\tilde{z}}$ in all cases, just like the pressure gradient term.

Reply:
However, I also suspect that @vanroekel will argue that there are no resolved stresses on the horizontal boundaries so the only terms we need to retain are the sub-grid-scale ones.